usaspending_api/disaster/v2/views/overview.py

```diff
@@ -196,5 +196,5 @@ def additional_totals() -> Optional[dict]:
                 "total_obligations": total_values["obligation_totals"] - defc_o_values["obligation_totals"],
                 "total_outlays": total_values["outlay_totals"] - defc_o_values["outlay_totals"],
             },
-            "total_budget_authority": total_values["total_budget_authority"] - defc_o_values["total_budget_authority"],
+            "total_budget_authority": total_values["obligation_totals"] - defc_o_values["obligation_totals"],
```

Comment:
I would completely remove the TBA section from the aggregates above. Since it is no longer being used there is no reason to keep it around.

Reply:
And also remove it from the values that are selected prior to the aggregate.